README.md

```diff
@@ -41,7 +41,7 @@ from this directory (the "nextstrain.org" directory).
 
 
 ### 3. Build the site
-`npm run build` runs `./build.sh` to build both the static site & an auspice client with customisations.
+`npm run build` runs `./build.sh` to build both the static site & an Auspice client with customisations.
 The following section details the different ways to serve these pages on a local server after building the site.
 
 ### 4. Run server
```

Comment:
Noting possibly the biggest dev change here: "server" now includes the static site and can't run without it.

Previously, `npm run server` would run just the server with Express.js routes, whereas now it will run along with a hot-reloading version of the static site. If you try to bypass that with `USE_PREBUILT_STATIC_SITE=1` and haven't built the static site, you'll get an error:

```
$ USE_PREBUILT_STATIC_SITE=1 node server.js
…
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '…/static-site/.next/BUILD_ID'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '…/static-site/.next/BUILD_ID'
}
```

I'm all good with the change, just wanted to note it. My guess for why the separation had existed with Gatsby is because building that version of the static site was somewhat slow, didn't allow hot-reloading, and needed workarounds on some machines. The Next.js build seems much faster and robust.

Reply:
👍 I've added in [6e97d3c](https://github.com/nextstrain/nextstrain.org/pull/810/commits/6e97d3cc3cf36aa3362c8629edc88b0957bfecba) which'll catch this (but only this) error and print a helpful message explaining that the static-site must be built.